includes/LinksUpdate.php

```diff
@@ -22,7 +22,7 @@
 
 use Wikia\Tasks\Tasks\BatchRefreshLinksForTemplate;
 
-class LinksUpdate {
+class LinksUpdate implements DeferrableUpdate {
```

Comment:
This will allow us to defer the update to the end of the request, so in FastCGI it won't block serving a response to the user.